test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```

Comment:
What is `require "test_helper"`?

Reply:
That means to load that library / module / file / class

Reply:
Why `test_helper`? for test only? Is there any helpers like `production_helper` etc?

Reply:
Er, cause this file is for you to write unit test. You need some basic methods to write your tests. Those are provided by the inbuilt test helper.

You can have production helper if you want. But it don't make sense. Where you going to put it and to do what? All your codes that are non test are by default for production already